cdp_scrapers/instances/seattle.py

```diff
@@ -311,10 +311,7 @@ def get_video_page_urls(
         # href attribute that contains videoid
 
         soup = BeautifulSoup(response, "html.parser")
-        for link in soup.find(
-            "div",
-            class_="paginationContainer",
-        ).find_all(
+        for link in soup.find("div", class_="paginationContainer",).find_all(
```

Comment:
Not sure why tox is modifying indentation in a couple of files. Do I need to modify any local settings?

Reply:
It's likely `black` the code formatter which `tox` runs.

Although thanks for pointing this out. I didn't know `cdp-scrapers` wasn't up-to-date. We ripped out `tox` in all the other libs in favor of a custom `Justfile`.

Sounds like I have a todo of upgrading the tooling in this repo to new cookiecutter version.

Reply:
Created a tracking issue in the repository: https://github.com/CouncilDataProject/cdp-scrapers/issues/126
 Feel free to add details or correct the description.